src/ocamlorg_frontend/components/secondary_footer.eml

```diff
@@ -1,30 +1,60 @@
 let render () =
-<footer class="bg-mild-contrast dark:bg-dark-mild-contrast border-t border-gray-200 pt-12 pb-20 md:p-0 md:fixed md:bottom-0 md:h-12 md:z-50 w-full" aria-labelledby="footer-heading">
-  <h2 id="footer-heading" class="sr-only">Footer</h2>
-  <div class="container-fluid wide">
-    <ul class="flex flex-col md:flex-row md:flex-wrap md:h-12 gap-x-6">
-      <li>
-        <a class="h-10 md:h-12 flex items-center mb-6 md:mb-0" href="<%s Url.index %>"><img class="h-8 md:h-6 pb-1" src="<%s Ocamlorg_static.Asset.url "logo-with-name.svg" %>" alt="OCaml"></a>
-      </li>
-      <li>
-        <a class="h-10 md:h-12 flex items-center text-lighter" href="<%s Url.getting_started %>" class="text-base leading-6 text-lighter">Get Started</a>
-      </li>
+  <footer class="flex flex-col gap-7 py-5 mt-6 border-solid border-t-[1px] border-[#00000033]">
+    <section class="flex justify-between items-end">
+      <div>
+        <a href="<%s Url.index %>"><img class="h-8" src="<%s Ocamlorg_static.Asset.url "logo-with-name.svg" %>" alt="OCaml"></a>
+        <p class="text-base leading-6 text-lighter">Innovation. Community. Security.</p>
+      </div>
+      <div class="flex items-center gap-4">
+        <a href="https://github.com/ocaml/ocaml"> <%s! Icons.github "w-6 h-6" %></a>
+        <a href="https://discord.gg/cCYQbqN"> <%s! Icons.discord "w-6 h-6" %></a>
+        <a href="https://twitter.com/ocamllang"><%s! Icons.twitter "w-6 h-6" %></a>
+        <a href="/feed.xml"><%s! Icons.rss "w-6 h-6" %></a>
+      </div>
+    </section>
 
-      <li>
-        <a class="h-10 md:h-12 flex items-center text-lighter" href="<%s Url.api %>" class="text-base leading-6 text-lighter">Standard Library API</a>
-      </li>
+    <section class="flex justify-between w-[80%]">
+      <div class="flex flex-col gap-4">
+        <div class="flex flex-col gap-4">
+          <h5 class="font-bold text-base leading-7 text-black">About</h5>
+          <a href="<%s Url.changelog %>" class="font-normal text-lighter leading-7">Changelog</a>
+          <a href="<%s Url.releases %>" class="font-normal text-lighter leading-7">Releases</a>
+          <a href="<%s Url.industrial_users %>" class="font-normal text-lighter leading-7">Industrial Users</a>
+          <a href="<%s Url.academic_users %>" class="font-normal text-lighter leading-7">Academic Users</a>
+          <a href="<%s Url.about %>" class="font-normal text-lighter leading-7">Why OCaml?</a>
+        </div>
 
-      <li>
-        <a class="h-10 md:h-12 flex items-center text-lighter" href="<%s Url.releases %>" class="text-base leading-6 text-lighter">Releases</a>
-      </li>
+      <div class="flex flex-col gap-4">
+        <h5 class="font-bold text-base leading-7 text-black">
+          Ecosystem & Community
```

Comment:
```suggestion
          Ecosystem
```

Since we had to change this heading to "Ecosystem" in the other footer, let's do it here as well. :slightly_smiling_face: